code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -350,13 +345,15 @@
 						UpdateName()
 						to_chat(user, "<span class='notice'>Name changed to [new_name].</span>")
 
+					// This option is now disabled. I had to comment out the front/side lines.
+					// It didn't work anyway, so nothing of value was lost!

```

Comment:
But it... huh. It does work, but it's literally the only thing you can change that gets updated when you examine the card. And examining the card breaks *entirely* if it's not in your hands, pockets, or ID slot exactly 3 seconds after it spawns into existence. I *guess* I could fix the examine contents never updating, and make this work properly...